code/src/App.js

```diff
@@ -1,9 +1,101 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import FormNewThought from './components/FormNewThought'
+import ThoughtsList from './components/ThoughtsList'
+import LoadingSpinner from './components/LoadingSpinner'
+import Header from 'components/Header'
+
+import { API_URL, LIKES_URL } from './utils/urls'
+
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([])
+  const [newThought, setNewThought] = useState('')
+  const [loading, setLoading] = useState(true)
```

Comment:
clever to put the loading to true in the beginning. Then you did not have to set it to true in the useEffect